.github/workflows/ci-action.yml

```diff
@@ -0,0 +1,24 @@
+name: GitHub Action Test
+
+on: [push]
+
+jobs:
+  action-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout current copy of rdme
+        uses: actions/checkout@v2.4.0
+        with:
+          path: rdme
+
+      - name: Checkout another repo containing OpenAPI file
+        uses: actions/checkout@v2.4.0
+        with:
+          repository: readmeio/oas-examples
+          path: external-repo
+          token: ${{ secrets.ACTION_PAT }}
+
+      - name: Run GitHub Action command test
+        uses: ./rdme/
+        with:
+          rdme: validate 3.1/json/petstore.json 
```

Comment:
Do we need to override the auth and login cache stuff in here to support passing that in with another `with` argument?

Reply:
Oh no I meant to replace the `rdme login` command and `--key` args that rdme has. How does rdme know what project to use?

Reply:
It doesn't need one for the `validate` command! That's in large part why I chose that one lol